### PR TITLE
Delivery for Manager - AGOL-2024.RO3 - Sprint1

### PR DESCRIPTION
**Pull request by sumitzarkar**
## Implemented
 - #769
 - #771
 - #784

_Note: This build is still under testing, we will share new build with the fixes if we found any issues in testing._

### CODE REVIEW

File: src/components/layer-table/layer-table.tsx [hunk 1]
```diff
@@ -1195,8 +1268,8 @@ export class LayerTable {
     skipControls: string[]
   ): void {
     let update = JSON.stringify(controlsThatFit) !== JSON.stringify(this._controlsThatFit);
-    const actionbar = document.getElementById("solutions-action-bar");
-    actionbar.childNodes.forEach((n: any) => {
+    const actionbar = document.getElementById?.("solutions-action-bar");
```
Comment (jmhauck) [inline src/components/layer-table/layer-table.tsx:1271]: @sumitzarkar I'll fix this before publish

File: src/components/map-card/map-card.tsx [hunk 2]
```diff
@@ -329,6 +476,13 @@ export class MapCard {
     );
   }
 
+  /**
+   * Called each time after the component is loaded
+   */
+    async componentDidRender(): Promise<void> {
```
Comment (jmhauck) [inline src/components/map-card/map-card.tsx:482]: @sumitzarkar does this really need to be set after each render?

**Review by jmhauck — APPROVED** [review]

File: src/components/crowdsource-manager/crowdsource-manager.tsx [hunk 3]
```diff
@@ -921,52 +921,210 @@ export class CrowdsourceManager {
...
+   * Should only occur on layer change
+   * 
+   * @protected
+   */
+  protected _initLayerExpressions(): void {
+    const layerExpressions = this._mapInfo?.filterConfig?.layerExpressions;
+    this._layerExpressions = layerExpressions ? layerExpressions.filter(
+      (exp) => exp.id === this._layer?.id) : [];
+    this._filterList.layerExpressions = this._layerExpressions;
+    this._layerExpressions.filter(lyrExp => {
```
[the remark this answers is not in the document]
Comment (sumitzarkar) [reply src/components/crowdsource-manager/crowdsource-manager.tsx:1089]: @jmhauck 
Now we are creating filter modal in crowdsource-manager.tsx file. So, when layer is ready we are fetching the layers expression and passing that in instant-apps-filter-list similar to what we did earlier in the layer-table

[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/crowdsource-manager/crowdsource-manager.tsx:1089]: @sumitzarkar I'm talking about the `filter` usage...would do nothing in this case

[the remark this answers is not in the document]
Comment (sumitzarkar) [reply src/components/crowdsource-manager/crowdsource-manager.tsx:1089]: @jmhauck - 

Got it, will remove this in next code drop

**Pull request merged**